pkg/kubelet/rkt/rkt.go

```diff
@@ -79,7 +79,7 @@ const (
 	k8sRktCreationTimeAnno           = "rkt.kubernetes.io/created"
 	k8sRktContainerHashAnno          = "rkt.kubernetes.io/containerhash"
 	k8sRktRestartCountAnno           = "rkt.kubernetes.io/restartcount"
-	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationMessagePath"
+	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/termination-message-path"
```

Comment:
Good question, I think it's ok to add hyphen now as it's not stable yet.
Ref #20728